survlearners/R/utils.R

```diff
@@ -97,168 +90,3 @@ sanitize_input = function(x,w,y,D) {
 	            y=y,
 	            D=D))
 }
-
-#' @title data simulation
-#'
-#' @description Generates a dataset of size \eqn{n} that can be used to experiment with the learners and meta-learners

```

Comment:
When making changes to places in R code that touches these Roxygen entries:
Remember to run Build->Document and commit the updated .Rd files

Reply:
You can add that to your workflow notes?

Reply:
yes, just figured it out, thanks